tests/integration/golang/fixtures/run.go

```diff
@@ -78,12 +94,44 @@ func (f RunFixtures) CreateRuns(
 		if err != nil {
 			return nil, err
 		}
+		tag := models.Tag{
```

Comment:
it is really, really hard to control over that in tests. for example you need to create different tags, metrics, params  with different parameters and then use them. right now it is impossible, because everything is hardcoded and the same for each test. Also you not always need to create `tags`, `metrics`, etc for each test. So it will be reasonable to create such data right in tests, where you can put any information you, information(parameters) could be different and you can see it.

Reply:
I don't think you want to fill each test with a lot of repetitive object creation -- better to have this in a helper and let the test focus on the subject of testing.